sdks/python/apache_beam/io/kafka.py

```diff
@@ -96,8 +96,16 @@
      ('commit_offset_in_finalize', bool), ('timestamp_policy', str)])
 
 
-def default_io_expansion_service():
-  return BeamJarExpansionService('sdks:java:io:expansion-service:shadowJar')
+def default_io_expansion_service(args = None):
+  extra_args = None
+  if args:
+    if type(args) != type({}):
```

Comment:
`isinstance(args, dict)`

Generally Python uses duck typing, so as long as this offers an item method (whether or not it's actually a dict) we should allow it through.

Reply:
Makes sense, thanks. It is a while since I did actually code in python. :)